claim/models.py

```diff
@@ -107,3 +129,24 @@ def update_map(cls, json_data):
         for polygon in json_data["features"]:
             polygon['claim_count'] = polygons_dict.get(
                 str(polygon["properties"]["ID"]), 0)
+
+
+class Polygon(models.Model):
+    # TODO(vegasq) looks like here we only link Organization with Layer.
+    # probably we should move coordinates and layer to Organization
+    # model? And remove this connectivity layer.
+    organization = models.ForeignKey(Organization, null=True)
```

Comment:
Polygon could cover not only building, but city district or county. So there would be polygons that are not organizations. Organization could be a subclass of Polygon